client/contracts/bindings/consensus/consensus.go

```diff
@@ -134,9 +124,9 @@ type consensus struct {
 
 // Binding returns a binding to the current consensus engine
 func Binding(contractBackend bind.ContractBackend, chainID *big.Int) (*consensus, error) {
-	addr, ok := mapValidatorMgrToAddr[chainID.Uint64()]
-	if !ok {
-		return nil, bindings.ErrNoAddress
+	addr, err := getAddressFromKNS("validatormgr.kowala", contractBackend)
```

Comment:
add "validatormgr.kowala" to the params package (ex: system_domains.go/domains.go)